docs/tracking-javascript-guide.md

```diff
@@ -693,6 +693,52 @@ _paq.push(['trackPageView']);
 
 ## Multiple Piwik trackers
 
+### If you use Piwik 2.16.2 or later
+
+It is possible to track your analytics data into multiple separate Piwik trackers. You may record your data either in the same Piwik server (into a different website ID) or you may record a duplicate of your data into another Piwik server altogether. 
```

Comment:
"into multiple separate Piwik trackers" is not really clear what is meant by "Piwik trackers". 

"It is possible to track your analytics data into either into a different website ID on the same server or you may record a duplicate of your data into another Piwik server altogether. "